plugins/quickinspector/quickinspector.cpp

```diff
@@ -799,6 +799,29 @@ void QuickInspector::pickElementId(const GammaRay::ObjectId &id)
         m_probe->selectObject(item);
 }
 
+QRectF QuickInspector::combinedChildrenRect(QQuickItem *object) const
+{
+    auto rect = object->childrenRect();
+
+    for (const auto child : object->childItems()) {
```

Comment:
put `object->childItems()` into a separate variable to avoid a detach warning
```c++
   const auto childItems = object->childItems();
   for (const auto child : childItems) {
```